api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export function authMiddleware(req, res, next) {
+  /* your code here */
+  const auth = req.get('Authorization');
+  const token = auth.split('Bearer ')[1];
```

Comment:
This will crash if there is no authorization header. Try sending a request without one and see if you get a 401 back. You probably get a 500.

Hint: You can fix this by adding a single `?` in here.